### ISSUE TITLE
Samplerate is set to 1sps when using Triggered Inspector

### ISSUE THREAD

**Issue by studiofuga** [BatchDrake/SigDigger#223]
When using the signal level triggered inspector, the sample rate is set to 1sps, while when using the pushbutton, it is correctly set to whatever the rate is.

Tested on HackRF One.

**Comment by studiofuga** [comment]
Confirmed the issue is in main and in 0.3.0 AppImage. 
Additionally, on my setup compiled from main, While manual Push to Capture works well, Autosqueltch triggers but doesn't capture anything.
Not if that is relevant, but there is a DC signal on all frequencies. On AppImage the same DC is waning.

**Comment by BatchDrake** [comment]
Confirmed, working on it.

**Comment by BatchDrake** [comment]
It should be fixed now, I've just triggered a new development build. Do you mind giving it a try?

**Comment by studiofuga** [comment]
Yeah, thanks, works like a charm. Tested right now on another linux box.

Out of couriosity, are there any plan to unify the three projects (sigutils, suscan, SuWidgets and SigDigger) in a single project wit submodules, perhaps porting all of them in cmake? I'd be happy to help on that eventually.

**Comment by studiofuga** [comment]
I'm using cmake in production code for qt projects pretty extensively, so I don't see any obvious difficulty. The only still unsolved issue is the packaging process, that's well solved on debian (dpkg-buildpackage is pretty straightforward) but still a mess in Windows and MacOS environment. But probably it's not an issue actually.
